docs/concepts/configuration/assign-pod-node.md

```diff
@@ -188,7 +188,7 @@ label having key "security" and value "S2".) See the [design doc](https://github
 for many more examples of pod affinity and anti-affinity, both the `requiredDuringSchedulingIgnoredDuringExecution`
 flavor and the `preferredDuringSchedulingIgnoredDuringExecution` flavor.
 
-As with node affinity, the legal operators for pod affinity and anti-affinity are `In`, `NotIn`, `Exists`, `DoesNotExist`, `Gt`, `Lt`.
+As with node affinity, the legal operators for pod affinity and anti-affinity are `In`, `NotIn`, `Exists`, `DoesNotExist`.
```

Comment:
As @aveshagarwal mentioned, node affinity supports "Gt" and "Lt" as well. Pod affinity term uses standard label selectors which support only In, NotIn, Exists, and DoesNotExist. So, please remove `As with node affinity, `.